core-web/libs/sdk/nextjs/src/lib/components/DotcmsLayout/DotcmsLayout.tsx

```diff
@@ -50,23 +51,18 @@ export function DotcmsLayout(props: DotcmsPageProps) {
     const router = useRouter();
     const pathname = usePathname();
 
-    const { rowsRef, isInsideEditor } = usePageEditor({
-        reloadFunction: router.refresh,
-        pathname
+    const client = sdkDotPageEditor.createClient({
+        onReload: router.refresh
     });
```

Comment:
maybe this method can be called `create`

Reply:
Also you need to be aware that this is going to be called in every render, we should manage this inside a `useEffect` or inside a `useMemo`. We need to provide a cleanup method to remove the event listeners, because if I'm not mistaken this is adding event listeners in every render, we need to be careful about that.